src/Bravo3/Orm/Drivers/Redis/RedisDriver.php

```diff
@@ -502,4 +511,42 @@ public function clearRefs($key)
     {
         $this->unit_of_work->addCommand('KeyDelete', [$key]);
     }
+
+    /**
+     * A wrapper function to wrap Redis commands which go to Predis Client
+     * in order to replay them if server connection issues occur.
+     *
+     * @param  string          $cmd function to call against the Predis client
+     * @param  array|callable  $params
+     * @param  int             $retry_iteration
+     * @return mixed
+     */
+    private function clientCmd($cmd, $params, $retry_iteration = 1) {
+        static $retry_delay = 200;
+
+        try {
+
+            if (1 > $retry_iteration) {
```

Comment:
This is broken.

Perhaps default $retry_iteration to 0 and do something like:

    $retry_delay = $this->initial_retry_delay * $retry_iteration;
    
    if ($retry_delay) {
        usleep($retry_delay * 1000);
    }

Currently, you are ALWAYS delaying, and your `if (1 > $retry_iteration) {` is around the wrong way. 

`$retry_delay` does not need to be static.